modules/smartadserverBidAdapter.js

```diff
@@ -44,17 +50,40 @@ export const spec = {
         appname: bid.params.appName && bid.params.appName != '' ? bid.params.appName : undefined,
         ckid: bid.params.ckId || 0,
         tagId: bid.adUnitCode,
-        sizes: bid.sizes.map(size => ({
-          w: size[0],
-          h: size[1]
-        })),
         pageDomain: utils.getTopWindowUrl(),
         transactionId: bid.transactionId,
         timeout: config.getConfig('bidderTimeout'),
         bidId: bid.bidId,
         prebidVersion: '$prebid.version$'
       };
 
+      const videoMediaType = utils.deepAccess(bid, 'mediaTypes.video');
+      if (!videoMediaType || (videoMediaType && videoMediaType.context === 'outstream')) {
+        // For banner and outstream, sizes are handled the same way.
+        var sizes = [];
+        if (videoMediaType && videoMediaType.context === 'outstream') {
+          sizes = videoMediaType.playerSize;
+        } else {
+          sizes = bid.sizes;
+        }
+        payload.sizes = sizes.map(size => ({
+          w: size[0],
+          h: size[1]
+        }));
+      } else if (videoMediaType && videoMediaType.context === 'instream') {
+        // Specific attributes for instream.
+        var playerSize = videoMediaType.playerSize[0];
+        payload.isVideo = true;
```

Comment:
This variable is causing the returned `outstream` bid from your adserver to be setup as a `banner` creative (see line 126 below).  The `outstream` bid should be treated as a `video` `mediaType` and ideally have an associated renderer.